apps/src/templates/sectionsRefresh/SectionsSetUpContainer.jsx

```diff
@@ -197,7 +197,9 @@ export default function SectionsSetUpContainer({
         recordSectionSetupEvent(section);
         // Redirect to the sections list.
         let redirectUrl = window.location.origin + '/home';
-        if (shouldShowCelebrationDialogOnRedirect) {
+        if (createAnotherSection) {
+          redirectUrl += '?openAddSectionDialog=true';
+        } else if (shouldShowCelebrationDialogOnRedirect) {
           redirectUrl += '?showSectionCreationDialog=true';
```

Comment:
can these be both true?

if that's the case, is this query built correctly? Can't you end up with something like
`'?openAddSectionDialog=true'?showSectionCreationDialog=true'`

instead of 
`'?openAddSectionDialog=true'&showSectionCreationDialog=true'`

Reply:
^The difference is the connecting character & vs ?

Reply:
We decided with product that we are not going to account for a case where they would both be true, so the 'if else' was intentional. The celebration screen currently shows as a 'final' step, and refactoring to show an eventual celebration screen is out of scope for this feature.

Reply:
Thank you for thinking of this!